### PR TITLE
added ntfs2ntfs command

### PR DESCRIPTION
**Pull request by datanel**

### CODE REVIEW

File: src/bin/ntfs2ntfs.rs [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+#[derive(Debug, StructOpt)]
+#[structopt(name = "ntfs2ntfs", about = "Convert an NTFS to an NTFS.")]
+struct Opt {
+    /// input directory.
+    #[structopt(short = "i", long = "input", parse(from_os_str), default_value = ".")]
+    input: PathBuf,
+
+    /// output directory
+    #[structopt(short = "o", long = "output", parse(from_os_str))]
+    output: PathBuf,
```
Comment (TeXitoi) [inline src/bin/ntfs2ntfs.rs]: Maybe we can put that optional, for just checking that everything is OK without needing to write in a directory?

File: src/bin/ntfs2ntfs.rs [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+}
+
+fn run() -> Result<()> {
+    info!("Launching ntfs2ntfs...");
+
+    let opt = Opt::from_args();
+
+    let objects = navitia_model::ntfs::read(opt.input)?;
+    navitia_model::ntfs::write(opt.output, &objects);
```
Comment (TeXitoi) [inline src/bin/ntfs2ntfs.rs]: There is no error handling in `ntfs::write`?

File: src/bin/ntfs2ntfs.rs (hunk 1 above)
Comment (datanel) [reply src/bin/ntfs2ntfs.rs]: ok

File: src/bin/ntfs2ntfs.rs (hunk 2 above)
Comment (datanel) [reply src/bin/ntfs2ntfs.rs]: no... weird

File: src/bin/ntfs2ntfs.rs [hunk 3]
```diff
@@ -44,7 +44,10 @@ fn run() -> Result<()> {
     let opt = Opt::from_args();
 
     let objects = navitia_model::ntfs::read(opt.input)?;
-    navitia_model::ntfs::write(opt.output, &objects);
+
+    if opt.output.is_some() {
+        navitia_model::ntfs::write(opt.output.unwrap(), &objects)?;
+    }
```
Comment (TeXitoi) [inline src/bin/ntfs2ntfs.rs]:
```rust
if let Some(ref output) = opt.output {
    navitia_model::ntfs::write(output, &objects)?;
}
```

File: src/ntfs/mod.rs [hunk 4]
```diff
@@ -126,34 +126,36 @@ pub fn read<P: AsRef<path::Path>>(path: P) -> Result<PtObjects> {
...
-    write::write_stops(path, &pt_objects.stop_points, &pt_objects.stop_areas);
-    write::write_comments(path, pt_objects);
-    write::write_codes(path, pt_objects);
-    write::write_object_properties(path, pt_objects);
+    )?;
+    write::write_calendar_and_calendar_dates(path, &pt_objects.calendars)?;
+    write::write_stops(path, &pt_objects.stop_points, &pt_objects.stop_areas)?;
+    write::write_comments(path, pt_objects)?;
+    write::write_codes(path, pt_objects)?;
+    write::write_object_properties(path, pt_objects)?;
```
Comment (TeXitoi) [inline src/ntfs/mod.rs]: :+1:

**Pull request merged**